src/Uno.UI/UI/Xaml/Controls/Flyout/FlyoutPopupPanel.cs

```diff
@@ -88,7 +88,8 @@ private Rect CalculateFlyoutPlacement(Size desiredSize)
 			}
 
 			var visibleBounds = ApplicationView.GetForCurrentView().VisibleBounds;
-			var anchorRect = anchor.GetAbsoluteBoundsRect();
+			var anchorTransform = (MatrixTransform)anchor.TransformToVisual(this);
```

Comment:
The goal of the previous PR was to remove this cast.

The bug **must** be fixed in `.GetAbsoluteBoundsRect()`.

We had a hard time in the past because of those casts... we must remove them.

Reply:
Agreed. Let's find why this happens first.

Reply:
What you probably need if this https://github.com/nventive/Uno/blob/master/src/Uno.UI/UI/Xaml/Media/GeneralTransform.cs#L16 to get `Flyout` location 

Note: There is also https://github.com/nventive/Uno/blob/master/src/Uno.UI/UI/Xaml/Media/GeneralTransform.cs#L31 but I think that you don't want to apply the transform on the `Flyout` (for instance if you have a `ScaleX=.1` you probably don't want to scale down your flyout ... to validate with Windows :) )

Reply:
![image](https://user-images.githubusercontent.com/32882405/59292893-f2799180-8c4b-11e9-9034-a0c4e0abedf5.png)

`.GetAbsoluteBoundsRect()` returns a rect with the render size of the button and the delta between the on-screen locations of the button (`thisRect`) and `Window.Current.Content` (`otherRect`). This causes a problem because the `Window.Current.Content` doesn't align with the `FlyoutPopupPanel`, since the latter takes visible padding into consideration.

Therefore, instead of using `Window.Current.Content` as reference point, we should use `FlyoutPopupPanel`.